### PR TITLE
fix docs blockstore formatting

### PR DESCRIPTION
**Pull request by KirillLykov**
#### Problem

Some formatting is broken

#### Summary of Changes

Fixed formatting

### CODE REVIEW

File: docs/src/validator/blockstore.md [hunk 1]
```diff
@@ -62,27 +62,28 @@ Repair requests for recent shreds are served out of RAM or recent files and out
...
+The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in. The entries will be sent on a channel exposed by the Blockstore.
 
-1. `fn get_slot_entries(slot_index: u64, entry_start_index: usize, max_entries: Option<u64>) -> Vec<Entry>`: Returns the entry vector for the slot starting with `entry_start_index`, capping the result at `max` if `max_entries == Some(max)`, otherwise, no upper limit on the length of the return vector is imposed.
+These subscription API's are as follows:
+
+1. `fn get_slots_since(slot_indexes: &[u64]) -> Vec<SlotMeta>`: Returns new slots connecting to any element of the list `slot_indexes`.
+
+2. `fn get_slot_entries(slot_index: u64, entry_start_index: usize, max_entries: Option<u64>) -> Vec<Entry>`: Returns the entry vector for the slot starting with `entry_start_index`, capping the result at `max` if `max_entries == Some(max)`, otherwise, no upper limit on the length of the return vector is imposed.
```
Comment (steviez) [inline docs/src/validator/blockstore.md:71]:
```suggestion
2. `fn get_slot_entries(slot: Slot, shred_start_index: u64) -> Result<Vec<Entry>>`: For the specified `slot`, return a vector of the available, contiguous entries starting from `shred_start_index`. Shreds are fragments of serialized entries so the conversion from entry index to shred index is not one-to-one. However, there is a similar function `get_slot_entries_with_shred_info()` that returns the number of shreds that comprise the returned entry vector. This allows a caller to track progress through the slot.
```

**Review by steviez — COMMENTED** [review]
I'm not mvines, but figured I could chime in here since I've worked with `Blockstore` extensively.

This document needs ALOT of work overall, but in any case, I think we may as well get the updates that you want to make right. And, I made myself a note to take a swing at performing the major overhaul that this document needs in the near future. Maybe the next time I'm on a plane with spotty-at-best internet capability or something.

File: docs/src/validator/blockstore.md [hunk 2]
```diff
@@ -62,27 +62,28 @@ Repair requests for recent shreds are served out of RAM or recent files and out
 
 ## Blockstore APIs
 
-The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in. The entries will be sent on a channel exposed by the Blockstore. These subscription API's are as follows: 1. `fn get_slots_since(slot_indexes: &[u64]) -> Vec<SlotMeta>`: Returns new slots connecting to any element of the list `slot_indexes`.
+The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in.
 
-1. `fn get_slot_entries(slot_index: u64, entry_start_index: usize, max_entries: Option<u64>) -> Vec<Entry>`: Returns the entry vector for the slot starting with `entry_start_index`, capping the result at `max` if `max_entries == Some(max)`, otherwise, no upper limit on the length of the return vector is imposed.
+These subscription API's are as follows:
```
Comment (steviez) [inline docs/src/validator/blockstore.md:67]: Given that these two lines don't have much technical detail in them, I think we can do away with the newlines between them. Ie, change
```
The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in.
 .
These subscription API's are as follows:
```
to
```
The Blockstore offers a subscription based API that ReplayStage uses to ask for entries it's interested in. These subscription API's are as follows:
```

**Review by steviez — APPROVED** [review]
Just the one nit about newlines.
- If you feel like changing it, I'll fire a quick ship it back at over since this one will be discarded
- If you don't feel like changing it, I'm not too torn up since as previously noted, this document is in need of some major work

File: docs/src/validator/blockstore.md (hunk 2 above)
Comment (KirillLykov) [reply docs/src/validator/blockstore.md]: done

**Review by steviez — APPROVED** [review]
LGTM

**Pull request merged**